.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Description
+<!--
+Include issue references (e.g., fixes [#issue](link))
+include check boxs to describe the tesps to do or have been done
+Include necessary implementation details, including clarifications, disclaimers etc, related to the approach used(e.g. I opted to use this algorithm because ... and test it in this way ...).
+-->
+
+### Changelog
+<!--
+Include changelog entry (e.g. Fixed an unexpected change in recenter button when resuming the app.).
+See https://github.com/mapbox/navigation-sdks/blob/main/documentation/android-changelog-guidelines.md.
+You can remove the changelog block and add a `skip changelog` label, when applicable.
+ -->
+ <changelog></changelog>
+
+### Screenshots or Gifs
+<!-- Include media files to provide additional context. It's REALLY useful for UI related PRs (e.g. ![screenshot gif](link)) -->
+
+
+<!--
+---------- CHECKLIST ----------
+1. Add related labels (`bug`, `feature`, `new API(s)`, `SEMVER-MAJOR`, `needs-backporting`, etc.).
+2. Update progress status on the project board.
+3. Request a review from the team, if not a draft.
+4. Add targeted milestone, when applicable.
+5. Create ticket tracking addition of public documentation pages entry, when applicable.
```

Comment:
These steps are only applicable to Mapbox employees. Maybe we should say so to avoid confusing external contributors.